src/schemas/686c-674/schema.js

```diff
@@ -0,0 +1,941 @@
+                  },
+                  birthDate: {
+                    $ref: '#/definitions/date',
+                  },
+                },
+              },
+            },
+          },
+        },
+      },
+
+      addChildPlaceOfBirth: {
+        type: 'object',
+        properties: {
+          childrenToAdd: {
+            type: 'array',
+            minItems: 1,
+            items: {
+              type: 'object',
+              properties: {
+                childPlaceOfBirth: {
+                  type: 'object',
+                  properties: {
+                    state: {
+                      $ref: '#/definitions/genericTextInput',
+                    },
+                    city: {
+                      $ref: '#/definitions/genericTextInput',
+                    },
+                  },
+                },
+                childStatus: {
+                  type: 'object',
+                  properties: {
+                    biological: {
+                      $ref: '#/definitions/genericTrueFalse',
+                    },
+                    adopted: {
+                      $ref: '#/definitions/genericTrueFalse',
+                    },
+                    notCapable: {
+                      $ref: '#/definitions/genericTrueFalse',
+                    },
+                    stepchild: {
+                      $ref: '#/definitions/genericTrueFalse',
+                    },
+                    dateBecameDependent: {
+                      $ref: '#/definitions/date',
+                    },
+                  },
+                },
+                'view:childStatusInformation': {
+                  type: 'object',
+                  properties: {},
+                },
+                childPreviouslyMarried: {
+                  type: 'string',
+                  enum: ['Yes', 'No'],
+                  default: 'No',
+                },
+
+                childPreviousMarriageDetails: {
+                  type: 'object',
+                  properties: {
+                    dateMarriageEnded: {
+                      $ref: '#/definitions/date',
+                    },
+                    reasonMarriageEnded: {
+                      type: 'string',
+                      enum: ['Divorce', 'Death', 'Annulment', 'Other'],
+                      default: 'Divorce',
+                    },
+                    otherReasonMarriageEnded: {
+                      $ref: '#/definitions/genericTextInput',
+                    },
+                  },
+                },
+              },
+            },
+          },
+        },
+      },
+
+      addChildAdditionalInformation: {
+        type: 'object',
+        properties: {
+          childrenToAdd: {
+            type: 'array',
+            minItems: 1,
+            items: {
+              type: 'object',
+              properties: {
+                doesChildLiveWithYou: {
+                  type: 'boolean',
```

Comment:
Should these booleans use the $ref for genericTrueFalse?

Reply:
There are quite a few changes like this that I ignored simply because it is more of a refactor, we should do this in a separate ticket.